influxdb/rootfs/etc/cont-init.d/influxdb.sh

```diff
@@ -17,3 +17,11 @@ if bashio::config.false 'reporting'; then
     sed -i 's/\<reporting-disabled\>.*/reporting-disabled=true/' /etc/influxdb/influxdb.conf
     bashio::log.info "Reporting of usage stats to InfluxData is disabled."
 fi
+
+# Disables external connectivity of InfluxDB
+if bashio::config.true 'influxd_local_only'; then
+    sed -i '/^\[http\]$/a\'$'\n''  bind-address = "127.0.0.1:8086"' /etc/influxdb/influxdb.conf
+else
+    bashio::log.warning 'External network connections to InfluxDB are allowed.'
+    bashio::log.warning 'Given that no SSL is used, this may be a concern.'
+fi
```

Comment:
Just disable the port in the add-on configuration settings if you don't want it exposed.

We don't need configuration options for that.

../Frenck

Reply:
Apologies, I should've probably depicted the exact usage scenario, for which this option was called for.

One of my HA setups is a supervised installation on a network not under my 100% control.  The InfluxDB add-on is quite useful there, however it's not desirable to let external machines on that network connect to it.  At the same time, using client tools from the host itself is quite essential there.

Deleting (disabling) the port number at which to export the HTTP port in the add-on configuration quite adequately prohibits the external port access, however it simultaneously prohibits the local port access too, which breaks the setup.

Adding this configuration option was what resolved the need.  Please reconsider merging it, given the extra detail.  If more detail should be added to the documentation for that, please let me know.

Thanks!